source/_dashboards/sections.markdown

```
@@ -77,17 +77,17 @@ In the sections view, you can rearrange sections and cards by dragging them to a
 2. To rearrange sections, hold the move {% icon "mdi:cursor-move" %} button and move the card.
 
-    <p class='img'>
-        <img src="/images/blog/2024-03-dashboard-chapter-1/drag-and-drop-sections.gif" alt="Rearranging sections by dragging"/>
-        Rearranging sections by dragging
-    </p>
+<p class='img'>
+  <img src="/images/dashboards/section_view_rearrange_sections.gif" alt="Rearranging sections by dragging"/>
+  Rearranging sections by dragging
+</p>
 
 3. To rearrange cards, tap and hold the card and move it to your desired location.
```

Markdownlint

Expected: 1; Actual: 3; Style: 1/1/1
Ordered list item prefix

(MD029, ol-prefix)